tutorials/11_ManagingRenv.md

```diff
@@ -1,71 +1,71 @@
 # basics
-REMIND uses [renv](https://rstudio.github.io/renv/) for managing required R packages. When starting R from the REMIND folder the corresponding renv is automatically activated. This means that packages are installed to and loaded from the renv subfolder instead of the usual package library. Thus updates in the global package library do not affect REMIND which is especially important on multi-user systems where packages may be loaded by some users while others are updating these packages. In such a situation the loaded packages are no longer valid and R crashes. Another advantage of renv is that it documents exactly which packages are currently installed, and it makes it easy to go back to a previous package configuration via `renv.lock` files.
+REMIND uses [renv](https://rstudio.github.io/renv/) for managing required R packages. When starting R from the REMIND folder the corresponding renv is automatically activated. This means that packages are installed to and loaded from the renv subfolder instead of the usual package library. Thus, updates in the global package library do not affect REMIND. This is especially important on multi-user systems where packages may be loaded by some users while others are updating these packages. In such a situation, the loaded packages are no longer valid and R crashes. Another advantage of renv is that it can document exactly which packages are currently installed, and it makes it easy to go back to a previous package configuration via renv.lock files.
```

Comment:
```suggestion
REMIND uses [renv](https://rstudio.github.io/renv/) for managing required R packages. When starting R from the REMIND folder the corresponding renv is automatically activated. This means that packages are installed to and loaded from the renv subfolder instead of the usual package library. Thus, updates in the global package library do not affect REMIND. This is especially important on multi-user systems where packages may be loaded by some users while others are updating these packages. In such a situation, the loaded packages are no longer valid and R crashes. Another advantage of renv is that it can document exactly which packages are currently installed, and it makes it easy to go back to a previous package configuration via `renv.lock` files.
```